regen-network/regen-web
# feat: Add Post Card component APP-49

## Description

Closes: #2265 

To test:




This PR adds the Post Card component to be used in the Project Timeline in the Project Page

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test



1. Open storybook and review UI
2. Manipulate controls tab to change behavior, check both desktop and mobile
3. Compare to Figma designs in linked issue
4. Review usage in Timeline

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
